tests/tests/eth_provider.rs

```diff
@@ -743,6 +747,11 @@ async fn test_send_raw_transaction(#[future] katana: Katana, _setup: ()) {
     // Assert the transaction hash and block number
     assert_eq!(tx.hash, transaction_signed.hash());
     assert!(tx.block_number.is_none());
+
+    // Assert that the number of pending transactions in the mempool has increased by 1
+    assert_eq!(eth_provider.mempool().unwrap().pool_size().pending, mempool_size.pending + 1);
+    // Assert that the transaction in the mempool exists
+    assert!(eth_provider.mempool().unwrap().get(&tx.hash).is_some());
```

Comment:
Nice test, can we check additionally here that the recovered hash is `tx.hash` (the hash that we want)?

Reply:
fixed, thanks